composer.json

```diff
@@ -100,5 +100,10 @@
         "psr-4": {
             "OpenEMR\\": "src"
         }
+    },
+    "autoload-dev": {
+        "psr-4": {
+            "OpenEMR\\": ["src", "tests"]
```

Comment:
You shouldn't need to repeat the src here (since the standard autoload stuff should also be included). Also another potential way to do this to avoid scanning places that don't contain PSR4 classes (the tests directory) could be:
```
"OpenEMR\\Tests\\": "tests\Tests"
```

Reply:
@bradymiller I've pushed updates for everything but the auto-load. I'll see how things go with Travis CI and will make this change this evening.